docs/docs/workflows/config.md

```diff
@@ -62,7 +62,22 @@ By default, Constellation uses `n2d-standard-4` VMs (4 vCPUs, 16 GB RAM) to crea
 </tabItem>
 <tabItem value="aws" label="AWS">
 
-By default, Constellation uses `m6a.xlarge` VMs (4 vCPUs, 16 GB RAM) to create your cluster. Optionally, you can switch to a different VM type by modifying **instanceType** in the configuration file. Supported are all nitroTPM-enabled machines with a minimum of 4 vCPUs (`xlarge` or larger). Refer to the [list of nitroTPM-enabled instance types](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enable-nitrotpm-prerequisites.html) or run `constellation config instance-types` to get the list of all supported options.
+By default, Constellation uses `m6a.xlarge` VMs (4 vCPUs, 16 GB RAM) to create your cluster.
+Optionally, you can switch to a different VM type by modifying **instanceType** in the configuration file.
+If you are using the default attestation type `awsSEVSNP` you can use the instance types described in [AWS's AMD SEV-SNP docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/snp-requirements.html).
+Please mind the region restrictions mentioned in the [Getting started](../getting-started/first-steps.md#create-a-cluster) section.
```

Comment:
not optimal to point from the more reference-like config page to the more guide-like page for further info

Reply:
Not sure what to do here. I don't want to copy the information as I assume the information will diverge at some point. But I also want to point the user to the limitations if they aren't aware. 
The workflows section also feels pretty guide-like to me?

Reply:
I'm also not sure what to do here. I thought maybe you would came up with an idea :D
Yes, reference-like is probably not correct. But I would feel more comfortable when the more-intro-level first-steps would reference the more-extensive-level config page. However, I also don't want to need to lookup something in the config workflow to be able to complete the first-steps.
So conclusion is probably to keep it as is.